dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/workflow/WorkflowTestUtil.java

```diff
@@ -46,6 +49,20 @@
     static final String ACTION_NAME_PREFIX = "action::";
     static final String CURRENT_STEP = "currentstep";
 
+    static final String SYSTEM_WORKFLOW = "System Workflow";
+    static final String DM_WORKFLOW = "Document Management";
+
+    static final String SAVE_AS_DRAFT = "Save as Draft";
+    static final String SEND_FOR_REVIEW = "Send for Review";
+    static final String SEND_TO_LEGAL = "Send to Legal";
+    static final String COPY = "Copy";
+    static final String SAVE = "Save";
+    static final String PUBLISH = "Publish";
+    static final String REPUBLISH = "Republish";
+    static final String UNPUBLISH = "Unpublish";
+    static final String ARCHIVE = "Archive";
+    static final String SAVE_PUBLISH = "Save / Publish";
+    static final String TWEET_THIS = "Tweet This!";
```

Comment:
Same as above, we can not depend of the Tweeter Action or Actionlet as will be remove it from the starter.

Reply:
removed.